lib/cmds/init.ts

```diff
@@ -16,21 +18,12 @@ export const builder = (yargs: Argv) => {
 
 export const init = async () => {
   greetings()
-  const { login } = await inquirer.prompt([
-    {
-      type: 'confirm',
-      name: 'login',
-      prefix: '🔒',
-      message: `You are ${chalk.red(
-        'not'
-      )} logged in, continue login in browser?`
-    }
-  ])
-  if (!login)
-    return console.log(
-      chalk.red('Please login to take advantage of contentful cli features!'),
-      `\nUse: ${chalk.green('contentful')} ${chalk.cyan('login')}`
-    )
+  const context = await getContext()
+  const managementToken = context.managementToken
+  await login({
+    context,
+    managementToken
+  })
```

Comment:
Could you elaborate on why you pass `context` and `managementToken` separately if you have `managementToken` in the context?

Reply:
maybe we don't need to call `login` at all if we already have `managementToken`, wdyt?

Reply:
Thanks, yes it's optional for the flag, so not needed for that case, updated 👍🏼